src/components/Entry/Card/index.tsx

```diff
@@ -146,21 +114,8 @@ const EntryCard = ({
         ) : (
           <Loading />
         )}
-        {showDescription &&
-          (desc && included ? (
-            <div className={css('new-card-description')}>
```

Comment:
Remove the bit of styling in the `style.css` file for the class `new-card-description`